src/main/java/edu/ucsb/cs156/gauchoride/config/SecurityConfig.java

```diff
@@ -88,6 +88,13 @@ private GrantedAuthoritiesMapper userAuthoritiesMapper() {
           if (email.endsWith("@ucsb.edu")) {
             mappedAuthorities.add(new SimpleGrantedAuthority("ROLE_MEMBER"));
           }
+          
+          Optional<User> u = userRepository.findByEmail(email);
+          if (u.isPresent() && u.get().getDriver()) {
+            mappedAuthorities.add(new SimpleGrantedAuthority("ROLE_DRIVER"));
+          } else {
+            mappedAuthorities.add(new SimpleGrantedAuthority("ROLE_RIDER"));
+          }
```

Comment:
Rider role should not be given by default. It needs to be given by an admin with an additional boolean field in the user table that specifies the rider status, like driver, at which point the user can be granted ROLE_RIDER

Reply:
FIxed in https://github.com/ucsb-cs156-s23/proj-gauchoride-s23-5pm-3/commit/b2b81791732a07bf4169aa6b7c91631264c13e6a

Reply:
awaiting #31 to fix X's